index.md

```diff
@@ -9,10 +9,10 @@ headerButtonUrl: "/what-is-scala/"
 
 # Links of the Download / API Docs sections
 gettingStarted:
-  mainTitle: "Download"
-  mainUrl: "/download"
-  subtitle: "Getting Started"
-  subtitleLink: "https://docs.scala-lang.org/getting-started.html"
+  mainTitle: "Getting Started"
+  mainUrl: "/documentation/getting-started"
+  subtitle: "download"
+  subtitleLink: "/download"
```

Comment:
I don’t understand this change, I’m not sure what `mainUrl` and `subtitleLink` are. We need to make sure that old URLs will still work after your changes.

Reply:
For me this change is important: the goal is to give more importance to the learning process, so first the `getting-started page` then the download page. Is that ok for you! 
![image](https://user-images.githubusercontent.com/7843237/106603785-0f6c8200-655f-11eb-8933-b95b22cbf8ed.png)

Reply:
the `documentation/getting-started` is a redirect .. but I can put the complete web address